pages/linux/setsid.md

```diff
@@ -0,0 +1,25 @@
+# setsid
+
+> Run a program in a new session if the calling process is not a process group leader.
+> The created session is by default not controlled by the current terminal.
+> More information: <https://manned.org/setsid>.
+
+- Run a program in a new session:
+
+`setsid {{program}}`
+
+- Run a program in a new session discarding the resulting output and error:
+
+`setsid {{program}} > /dev/null 2>&1`
```

Comment:
```suggestion
`setsid {{program}} > &> /dev/null`
```

Reply:
I initially written like this, but when i tested it in `dash` it didn't work, so i opted for the most shell agnostic and maybe POSIX compliant way (if i understood [this](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_02) well).

Reply:
Just as a curiosity, neither seemed to work with `[t]csh`...

Reply:
Yes, maybe it's better to leave your code unchanged because `Tl;Dr` is not restricted to concrete shells.